src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -0,0 +1,410 @@
+
+### Development environment
+
+During the development you'll be able to use sensors through `localhost`. The simplest way is to
+serve your web application using [Web Server for Chrome](https://chrome.google.com/webstore/detail/web-server-for-chrome/ofhbbkphhbklhfoeikjpcbhemlocgigb).
+If you are developing for mobile device, set up [port forwarding](/web/tools/chrome-devtools/remote-debugging/local-server)
+for your local server, and you are ready to rock!
+
+When your code is ready, you would need to deploy it on a server that supports HTTPS.
+[GitHub Pages](https://pages.github.com/) are served over HTTPS, making it a great place to share
+your demos.
+
+Note: Don't forget to enable [Generic Sensor API](#generic-sensor-api-in-chrome) in Chrome.
+
+### 3D model rotation
+
+In this simple example, we use the data from an absolute orientation sensor to
+modify rotation quaternion of a 3D model. The <code>model</code> is a three.js
+<code>[Object3D](https://threejs.org/docs/index.html#api/core/Object3D)</code> class instance
+that has <code>[quaternion](https://threejs.org/docs/index.html#api/core/Object3D.quaternion)</code>
+property. The following code snippet from the
+ [orientation phone](https://github.com/intel/generic-sensor-demos/tree/master/orientation-phone)
+demo, illustrates how absolute orientation sensor can be used to rotate 3D model.
+
+    function initSensor() {
+        sensor = new AbsoluteOrientationSensor({frequency: 60});
+        sensor.onreading = () => model.quaternion.fromArray(sensor.quaternion);
+        sensor.onerror = event => {
+            if (event.error.name == 'NotReadableError') {
+                console.log("Sensor is not available.");
+            }
+        }
+        sensor.start();
+    }
+
+Device's orientation will be reflected in 3D <code>model</code> rotation within the WebGL scene.
+
+<figure align="center">
+  <img src="/web/updates/images/2017/09/sensors/orientation_phone.png"
+    alt="Sensor updates 3D model's orientation">
+  <figcaption><b>Figure 4</b>: Sensor updates orientation of a 3D model</figcaption>
+</figure>
+
+### Punchmeter
+
+The following code snippet is extracted from the
+[punchmeter demo](https://github.com/intel/generic-sensor-demos/tree/master/punchmeter),
+illustrating how linear acceleration sensor can be used to calculate the maximum
+velocity of a device under the assumption that it is initially laying still.
+
+```
+   this.maxSpeed = 0;
+   this.vx = 0;
+
+   this.ax = 0;
+   this.t = 0;
+
+   function onreading() {
+     let dt = (this.accel.timestamp - this.t) * 0.001; // In seconds.
+     this.vx += (this.accel.x + this.ax) / 2 * dt;
+
+     let speed = Math.abs(this.vx);
+
+     if (this.maxSpeed < speed)
+       this.maxSpeed = speed;
+
+     this.t = this.accel.timestamp;
+     this.ax = this.accel.x;
+   }
+   ....
+
+   this.accel.addEventListener('reading', onreading);
+
+```
+
+The current velocity is calculated as an approximation to integral of the
```

Comment:
'an approximation to integral'

If I understand the sentence correctly, this should be 'an approximation of the integral'.

Reply:
I think mathematically correct would be 'an approximation to the integral'. I would add 'the' before integral.

https://en.wikipedia.org/wiki/Numerical_integration